transactions/scripts/get_topshot_metadata.cdc

```diff
@@ -0,0 +1,147 @@
+import TopShot from 0xTOPSHOTADDRESS
+import MetadataViews from 0xMETADATAVIEWSADDRESS
+
```

Comment:
Why define a new struct here when you can just use and return TopShot.TopShotMomentMetadataView?

You could just delete lines 4-100, delete lines 115-146, and just return the view from the contract.

Reply:
Ah, yes. Thank you Josh. This is fixed now.